mail/urlscan/Portfile

```diff
@@ -4,11 +4,11 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        firecat53 urlscan 0.9.5
+github.setup        firecat53 urlscan 1.0.3
 categories          mail python
-platforms           darwin
```

Comment:
does this compile stuff and/or install architecture-dependent files? If jot it should set:

platforms {darwin any}
supported_archs arch

Reply:
For the architectures, should it rather be:
supported_archs     noarch

Reply:
sorry, indeed that's what I meant 😉